--- a/src/detail/standalone/macos/auv3/AUv3HostAppDelegate.mm
+++ b/src/detail/standalone/macos/auv3/AUv3HostAppDelegate.mm
@@ -243,6 +243,7 @@
   switch ([AVCaptureDevice authorizationStatusForMediaType:AVMediaTypeAudio])
   {
     case AVAuthorizationStatusNotDetermined:
+    {
       [AVCaptureDevice requestAccessForMediaType:AVMediaTypeAudio
                                completionHandler:^(BOOL granted) {
                                  if (!granted) return;
@@ -258,6 +259,7 @@
                                  });
                                }];
       break;
+    }
     default:
       break;
   }
